project/BuildHelper.scala

```diff
@@ -12,6 +12,10 @@ object BuildHelper {
     incOptions ~= (_.withLogRecompileOnMacro(false))
   )
 
+  val withScala3 = Seq(
+    crossScalaVersions       := Seq(Scala212, Scala213, Scala3),
+  )
+
```

Comment:
Instead of this, could you please bring zio-telemetry to the same standard as other projects and load scala versions from actions file? Basically you need to do the following:

- [add snakeyaml plugin](https://github.com/zio/zio-redis/pull/385/files#diff-8fd74420ca55aa91d4c53775d2a55a999c405b5a4fafeaf73b7f0ef83e1f3b0cR11)
- [define versions map](https://github.com/zio/zio-redis/pull/385/files#diff-dd3701b993fcc974a9a6d017159aac00d4b41bcd05ad319815c6001311c669d5R8)

We introduced this change after zio-telemetry was ported to Github Actions and we didn't update it :man_facepalming: .

Reply:
We already have the project map - https://github.com/zio/zio-telemetry/blob/master/project/BuildHelper.scala#L15-L26, I did it so that we don't build example projects for Scala3

Reply:
Oh sorry, good point. I would just swap the two - `stdOptions` should include Scala 3, and we should be explicit about missing Scala 3 in examples.

Reply:
@mijicd good point, I swapped it. Also I bumped scala version to 2.12.15